eng/templates/stages/release.yml

```diff
@@ -148,6 +152,33 @@ stages:
             draft='--draft'
           fi
 
+          git_user=dotnet-bot
+          git_email=dotnet-bot@microsoft.com
+
+          git config --global user.name "$git_user"
+          git config --global user.email "$git_email"
+
+          vmr_url=$(echo $vmrPublicUrl | sed "s,https://.*@,https://$(git_user):${GH_TOKEN}@,g")
+
+          git fetch $vmr_url tag $releaseTag --no-tags
```

Comment:
Have you tested this? Seems like this command will kill the execution because we have `set -euo pipefail` so it will never reach the if?

Reply:
I did, but without the `set -euo pipefail`. Thanks for pointing it out

Reply:
What would you suggest to do in this case actually? There is also a `set +x` below and I don't think wrapping the code with `set +e` and `set -e` is the way to go

Reply:
That should not stop the script

Reply:
Did that + used `set +e` since there were other commands later that could fail the job